spec/datetimeformat.html

```diff
@@ -186,6 +186,19 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
             1. Let _rangePatterns_ be _bestFormat_.[[rangePatterns]].
         1. Set _dateTimeFormat_.[[Pattern]] to _pattern_.
         1. Set _dateTimeFormat_.[[RangePatterns]] to _rangePatterns_.
+        1. Let _nfOptions_ be OrdinaryObjectCreate(*null*).
+        1. Perform ! CreateDataPropertyOrThrow(_nfOptions_, *"useGrouping"*, *false*).
+        1. Set _dateTimeFormat_.[[NumberFormat1]] to ? Construct(%NumberFormat%, &laquo; _locale_, _nfOptions_ &raquo;).
```

Comment:
There is no `_locale_` here, you should use either `_r_.[[locale]]` or `_dateTimeFormat_.[[Locale]]`, preferably the latter IMO.